### PR TITLE
feat: diable  button on table contributors when no budget left

### PR DESCRIPTION
**Pull request by tdelabro**

### CODE REVIEW

File: frontend/src/components/ContributorsTable/index.tsx [hunk 1]
```diff
@@ -22,10 +22,14 @@ import {
   ProjectDetailsActionType,
   ProjectDetailsDispatchContext,
 } from "src/pages/ProjectDetails/ProjectDetailsContext";
+import { group } from "yargs";
+
+const MIN_SENDABLE_BUDGET = 63;
```
Comment (Bernardstanislas) [inline frontend/src/components/ContributorsTable/index.tsx:27]: I would rather get this value with https://github.com/onlydustxyz/marketplace/blob/main/frontend/src/hooks/useWorkEstimation.ts#L30

File: frontend/src/components/ContributorsTable/index.tsx [hunk 2]
```diff
@@ -22,10 +22,14 @@ import {
   ProjectDetailsActionType,
   ProjectDetailsDispatchContext,
 } from "src/pages/ProjectDetails/ProjectDetailsContext";
+import { group } from "yargs";
```
Comment (Bernardstanislas) [inline frontend/src/components/ContributorsTable/index.tsx:25]: Not used

File: frontend/src/translations/en.json [hunk 3]
```diff
@@ -8,7 +8,8 @@
       "leftToPay": "Left to Pay",
       "paidContributions": "Paid contributions",
       "totalEarned": "Total earned",
-      "userRegisteredTooltip": "User has linked their OnlyDust profile"
+      "userRegisteredTooltip": "User has linked their OnlyDust profile",
+      "noBudgetLeft": "There's no more budget for this project, please contact Only Dust to ask for a refill."
```
Comment (Bernardstanislas) [inline frontend/src/translations/en.json:12]: "Only Dust" => "OnlyDust"

**Review by Bernardstanislas — COMMENTED** [review]
It looks like the schema generation removed too much things, make sure your local Hasura is up to date before running the schema generate command
+ other comments in the review

File: frontend/src/components/ContributorsTable/index.tsx (hunk 1 above)
Comment (tdelabro) [reply frontend/src/components/ContributorsTable/index.tsx:27]:
```
const rates = {
  [Steps.Days]: DAY_RATE_USD,
  [Steps.Hours]: DAY_RATE_USD / (maxSteps[Steps.Hours] + 1),
}
```
How do you get the value you want ?
63 = 500 /7 = ?

Comment (tdelabro) [reply frontend/src/components/ContributorsTable/index.tsx:26]: rates.hours ?

Comment (tdelabro) [reply frontend/src/components/ContributorsTable/index.tsx]: done

File: frontend/src/translations/en.json (hunk 3 above)
Comment (tdelabro) [reply frontend/src/translations/en.json]: fixed

**Review by Bernardstanislas — APPROVED** [review]

**Pull request merged**